src/components/selectable/selectable_list/selectable_list_item.tsx

```diff
@@ -197,25 +225,40 @@ export class EuiSelectableListItem extends Component<
       }
     }
 
+    const instructions = (instruction || state) && (
+      <EuiScreenReaderOnly>
+        <div>
+          {state}
+          {instruction}
```

Comment:
One more thing that just occurred to me - our individual state/instruction strings don't have spaces between them, which **does** matter in the cadence that screen readers read sentences out as (VO is pretty rushed right now reading this out), so we should make sure to manually include it!

```suggestion
          {state}
          {state && instruction ? ' ' : null}
          {instruction}
```

Or maybe if we want to get fancy?...
```suggestion
          {[state, instruction].join(' ')}
```
lol this one is probably a bad idea 🙈

Reply:
Actually there's a lack of spacing between the name and the state as well. Can we add some kind of punctuation for this?
```suggestion
          {' - '} // Adds a breath/screen reader break between reading out the item name and state
          {state}
          {state && instruction ? ' ' : null}
          {instruction}
```

Reply:
Actually super interesting, sorry - I was testing VO on Firefox when I made this comment, it looks like Safari doesn't have this issue (although Chrome also does have the same rushed reading as well). Feel free to disregard this comment as optional if you'd rather not optimize for FF/Chrome, as I know VO+Safari is the main Mac SR+browser combo, currently at least 🤷‍♀️ 

However it might be worth noting that FF on VO is how I noticed `aria-selected` wasn't working - VO was reporting out "selected" for all options even when it wasn't actually selected (whereas it doesn't do that on Safari for some interesting reason).